src/content/docs/merchants/storefront-builder/visual-editor.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Using the Visual Editor
+description: Learn about the Visual Editor and how to use it to edit your storefront content.
+---
+
+The **Visual Editor** is powered by Adobe's Universal Editor and uses many of the same features as the Universal Editor. It is a powerful visual editing tool that enables business users and content authors to edit website content in real-time with immediate live preview. This in-context editing experience eliminates the need to switch between different tools or wait for publishing cycles to see changes.
+
+Visual Editor provides a WYSIWYG (What You See Is What You Get) experience while updating content directly on the rendered page of your storefront. It offers an intuitive interface for updating text, images, blocks, and drop-ins without requiring technical knowledge or switching between different tools.
+
+With Visual Editor, you can make immediate changes to your storefront content and see the results instantly, eliminating the need for separate preview environments or publishing cycles. This tool is particularly valuable for marketing teams, content creators, and business users who need to quickly update product descriptions, promotional content, or page layouts to respond to market demands or campaign requirements.
+
+## What is the Visual Editor?
+
+Visual Editor is a subset of features from the AEM Universal Editor that provides business users and content authors with an intuitive, visual interface for content management. As a key component of the Storefront Builder ecosystem, it bridges the gap between technical implementation and content authoring.
+
+### Key Features
+
+The Visual Editor offers several powerful capabilities:
+
+- **Real-time editing**: Make changes and see them immediately on the live page
+- **In-context editing**: Edit content directly where it appears on the storefront
+- **WYSIWYG interface**: See exactly how your content will look to customers
+- **Block and drop-in management**: Easily update both content and commerce components
+- **No technical knowledge required**: Business users can make changes without developer assistance
+- **Seamless workflow**: Eliminate the need to switch between different editing tools
+
+### Integration with Storefront Builder
+
+Visual Editor works seamlessly with other Storefront Builder components:
+
+- **Document Authoring (DA.live)**: Edit content that's managed and organized in the CMS
+- **Content and Commerce Blocks**: Visually modify both static content blocks and dynamic commerce blocks
+- **Drop-ins**: Update commerce-specific components with real-time preview
```

Comment:
This line seems redundant, as you are really updating "Commerce Blocks." Drop-in changes are proxied through the Commerce Block.